<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="jadx.tests.integration.trycatch.TestTryCatch4" tests="1" skipped="0" failures="0" errors="0" timestamp="2016-04-22T15:41:02" hostname="box723" time="0.037">
  <properties/>
  <testcase name="test" classname="jadx.tests.integration.trycatch.TestTryCatch4" time="0.037"/>
  <system-out><![CDATA[15:41:02 INFO  - converting to dex: jadx-tmp-474732610570634743084930630213199.jar ...
15:41:02 INFO  - loading ...
15:41:02 DEBUG - Load: /tmp/jadx-tmp-474732610570634743084930630213199.jar
15:41:02 DEBUG - '.arsc' file not found
15:41:02 WARN  - Unknown 'R' class, create references to 'R'
15:41:02 DEBUG -  Duplicated block: B:7:0x0015 in jadx.tests.integration.trycatch.TestTryCatch4$TestCls.test(java.lang.Object):java.lang.Object
-----------------------------------------------------------
package jadx.tests.integration.trycatch;

import java.io.File;
import java.io.FileNotFoundException;
import java.io.FileOutputStream;

public class TestTryCatch4$TestCls {
    private Object test(Object obj) {
        FileOutputStream fileOutputStream;
        try {
            FileOutputStream output = new FileOutputStream(new File("f"));
            try {
                fileOutputStream = output;
                return new Object();
            } catch (FileNotFoundException e) {
                fileOutputStream = output;
                System.out.println("Exception");
                return null;
            }
        } catch (FileNotFoundException e2) {
            System.out.println("Exception");
            return null;
        }
    }
}
-----------------------------------------------------------
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
